content/docs/querying/basics.md

```diff
@@ -85,7 +85,7 @@ For example, this selects all `http_requests_total` time series for `staging`,
     http_requests_total{environment=~"staging|testing|development",method!="GET"}
 
 Label matchers that match empty label values also select all time series that do
-not have the specific label set at all. Regex-matches are fully anchored.
+not have the specific label set at all. Regex-matches are not anchored.
```

Comment:
I'd say 99% of the use of "partial match" in the context of regular expressions means "the whole regular expression matches a part of the target string". But when doing my little research about what the standard anchoring behavior is, I found a few occurrences (namely in Boost) that use "partial match" as in "parts of the regexp matches". So I agree we should just talk about anchoring to avoid misunderstandings.